ArcGISRuntimeSDKQt_CppSamples/Scenes/ChangeAtmosphereEffect/ChangeAtmosphereEffect.qrc

```diff
@@ -0,0 +1,11 @@
+<RCC>
+    <qresource prefix="/Samples/Scenes/ChangeAtmosphereEffect">
+        <file alias="sample.json">ChangeAtmosphereEffect.json</file>        
+        <file>ChangeAtmosphereEffect.qml</file>
+        <file>ChangeAtmosphereEffect.h</file>
+        <file>ChangeAtmosphereEffect.cpp</file>
+		<file>main.qml</file>
```

Comment:
alignment looks off here

Reply:
I think may be a template issue as well.